### ISSUE TITLE
[Bug] Label layout error while zoom

### ISSUE THREAD

**Issue by li942475402** [apache/echarts#18320]
### Version

5.4.1

### Link to Minimal Reproduction

https://echarts.apache.org/examples/zh/editor.html?c=graph-label-overlap

### Steps to Reproduce

1. set normal label position: `right`
2. set select label position: `inside`
3. zoom graph

### Current Behavior

select label position is `right`

### Expected Behavior

select label  position should be `inside`

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by helgasoft** [comment]
not-a-bug.  Zoom does not make any selection. Also not to forget to **enable** select with _selectedMode_:
```
        selectedMode: true,
        select: {label: { position: 'inside'}},
```
Then just **click** on any node to **select** it and see label inside.

**Comment by li942475402** [comment]
> 不是错误。缩放不会进行任何选择。另外不要忘记**使用**_选定模式_启用选择：
> 
> ```
>         selectedMode: true,
>         select: {label: { position: 'inside'}},
> ```
> 
> 然后只需**单击**任何节点即可**选择**它并查看其中的标签。

Sorry I missed some steps, please try again after modification, you should see the problem

**Comment by helgasoft** [comment]
yes, I see it now - label of selected does not stay inside on zoom. One must hover with mouse to put label back in.
Could be considered a bug 🐛
